airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -14,210 +14,282 @@
 #
```

Comment:
Thanks for reviewing my comments @Fokko. I will just make the changes mentioned in [this PR](https://github.com/apache/incubator-airflow/pull/2703). I will add tests and make a little more modification to handle if both a table and a query for an import are passed and what to do then.

Reply:
After a quick check Sqoop raises an error. I will do the same and throw an exception for this in Airflow